src/Entity/ProductCategory.php

```diff
@@ -41,6 +43,16 @@ class ProductCategory
      */
     private $modificationDate;
 
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Product", mappedBy="category")
+     */
+    private $product;
```

Comment:
This variable will contain an array of products. Please rename it to $products. Remember to change the code in every place where it is used.